docs/user-guide/kubectl/kubectl_convert.md

```diff
@@ -0,0 +1,101 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<h1>PLEASE NOTE: This document applies to the HEAD of the source
+tree only. If you are using a released version of Kubernetes, you almost
+certainly want the docs that go with that version.</h1>
+
+<strong>Documentation for specific releases can be found at
+[releases.k8s.io](http://releases.k8s.io).</strong>
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+## kubectl convert
+
+Convert config files between different API versions
+
+### Synopsis
+
+
+Convert config files between different API versions.
+The command takes filename or stdin as input, and convert it to the format of version
+specified by -v flag. If target version is not specified, convert to latest version.
+Both YAML and JSON formats are accepted.
+
+
+```
+kubectl convert -i FILENAME
+```
+
+### Examples
+
+```
+// Convert 'pod.yaml' to latest version and print to stdout.
+$ kubectl convert -i pod.yaml
+
+// Convert 'pod.yaml' to version v1beta3 and output to 'b3pod.yaml'.
+$ kubectl convert -i pod.yaml -v v1beta3 -o b3pod.yaml
+
+// Convert 'pod.yaml' to latest version and rewrite to 'pod.yaml'.
+$ kubectl convert -r pod.yaml
+
+```
+
+### Options
+
+```
+  -h, --help=false: help for convert
+  -i, --input="-": Input source; '-' means stdin.
```

Comment:
OK, I'll try this out. Besides I think use stdin is meaningless. If user would like to type the template, why not write in the version they want directly but in other versions and do a convertion. Stdin is kept because it does no harm, but if most people agree it's meaningless, I would like to remove it.